tests/testthat/test-add_gloabl_p.tbl_uvregression.R

```diff
@@ -0,0 +1,52 @@
+skip_on_cran()
```

Comment:
Can you add a test for the `geeglm` package model? This class of model as a separate S3 method for the `anova_fun= global_pvalue_fun`

Reply:
I plugged in the geeglm example that they have on their own help page but it gives a buncha messages/warnings, do you have a better example/clinical trial based one in mind?